front/src/Api/iframe/ui.ts

```diff
@@ -109,11 +109,33 @@ export class WorkAdventureUiCommands extends IframeApiContribution<WorkAdventure
         sendToWorkadventure({
             type: "registerMenuCommand",
             data: {
-                menutItem: commandDescriptor,
+                menuItem: commandDescriptor,
             },
         });
     }
 
+    registerMenuIframe(menuName: string, iframeUrl: string) {
+        sendToWorkadventure({
+            type: "registerMenuIframe",
+            data: {
+                name: menuName,
+                url: iframeUrl,
+            },
+        });
+    }
```

Comment:
Rather than adding a separate method, we should probably aim for something more generic.

Like:

```
registerMenuCommand(commandDescriptor: string, options: {
    callback?: (commandDescriptor: string) => void,
    menuIframe?: string,
    autoClose?: boolean, // to know if we should close the menu by default or not
})
```

Also, the `registerMenuCommand` might return a "Menu" class with a "close()" method (rather than exposing a "unregisterMenu" method.

The hard part is that we must keep compatibility with the old `registerMenuCommand` function (that we will deprecate).
This means we need to use function overloading: https://www.typescriptlang.org/docs/handbook/functions.html#overloads